plugins/org.jboss.tools.openshift.express.ui/src/org/jboss/tools/openshift/express/internal/ui/explorer/ExpressExplorerContentProvider.java

```diff
@@ -25,7 +30,34 @@
  * Content Provider for OpenShift 2 content
  */
 public class ExpressExplorerContentProvider extends BaseExplorerContentProvider {
-
+	private ConnectionsRegistry input;
+	private IConnectionsRegistryListener connectionListener = new ConnectionsRegistryListener();
+	private static final Collection<String> PROPERTY_BLACKLIST = Collections.unmodifiableList(Arrays.asList("token"));
```

Comment:
could we use some constant (I see it living as private constant in Connection, maybe we should have it public and moved to IOpenShiftConnction?) since we here want to black list some notification that's being sent out by someone and we might get out-of-sync if the notifier gets refactored?